spec/models/like_spec.rb

```diff
@@ -13,6 +13,23 @@
   it 'has a valid factory' do
     expect(FactoryGirl.build(:like)).to be_valid
   end
+  
+  describe '#participation' do
+    before do
+      @like = alice.like!(@status)
+    end
+
+    it 'has participation' do
+      participations = Participation.where(target_id: @like.target_id, author_id: @like.author_id)
+      expect(participations.count).to eq(1)
+    end
+    
+    it 'destroy participation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.